webapp/pages/profile/_id/_slug.vue

```diff
@@ -507,7 +460,11 @@ export default {
         return UserQuery(this.$i18n)
```

Comment:
> [<img alt="alina-beck" height="40" width="40" align="left" src="https://avatars1.githubusercontent.com/u/38261864?s=88&v=4">](https://github.com/alina-beck) **Authored by [alina-beck](https://github.com/alina-beck)**
_<time datetime="2020-04-07T14:12:26Z" title="Tuesday, April 7th 2020, 4:12:26 pm +02:00">Apr 7, 2020</time>_

---
> **Outdated (history rewrite)** - original diff
---
```diff
@@ -483,6 +433,9 @@ export default {
       this.user.followedByCurrentUser = followedByCurrentUser
       this.user.followedBy = followedBy
     },
+    fetchAllConnections(type) {
+      this[`${type}Count`] = Infinity
```


I like this cause it's elegant – but it's also prone to errors because it will not be found when refactoring the variable names of the counts or values of the `type` variable, so I think I'd rather go with the slightly longer but safer solution:
```
if (type === 'following') this.followingCount = Infinity
if (type === 'followedBy') this.followedByCount = Infinity
```

What do you think?

Reply:
> [<img alt="rbeer" height="40" width="40" align="left" src="https://avatars1.githubusercontent.com/u/38261864?s=88&v=4">](https://github.com/rbeer) **Authored by [rbeer](https://github.com/rbeer)**
_<time datetime="2020-04-09T21:17:17Z" title="Thursday, April 9th 2020, 11:17:17 pm +02:00">Apr 9, 2020</time>_

---


2c7ca87ac